tests/dummy/app/controllers/table-height.js

```diff
@@ -0,0 +1,64 @@
+// BEGIN-SNIPPET table-height-controller
```

Comment:
Its for the fancy code snippets in the demo page (like in http://opensource.addepar.com/ember-table/#/simple)